### PR TITLE
Drama.gg Presence

### PR DESCRIPTION
**Pull request by toki0179**

### CODE REVIEW

File: websites/D/Drama/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Drama",
+    "author": {
+      "name": "Toki",
+      "id": "401067750354845727"
+    },
+    "category": "socials & stream",
+    "logo": "https://cdn.discordapp.com/icons/730015520568639509/a_af5921a891d60ff6b22c79f0d44c7e3b.webp?size=512",
```
Comment (VocalZero) [inline websites/D/Drama/dist/metadata.json:9]: You need to upload this image to Imgur.

File: websites/D/Drama/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Drama",
+    "author": {
+      "name": "Toki",
+      "id": "401067750354845727"
+    },
+    "category": "socials & stream",
+    "logo": "https://cdn.discordapp.com/icons/730015520568639509/a_af5921a891d60ff6b22c79f0d44c7e3b.webp?size=512",
+    "thumbnail": "https://zyrex.xyz/storage/a2d579e2-55f2-44a3-81fc-bcae0ff389a6.png",
```
Comment (VocalZero) [inline websites/D/Drama/dist/metadata.json:10]: You need to upload this image to Imgur.

File: websites/D/Drama/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Drama",
+    "author": {
+      "name": "Toki",
+      "id": "401067750354845727"
+    },
+    "category": "socials & stream",
```
Comment (VocalZero) [inline websites/D/Drama/dist/metadata.json:8]: This category does not exist in PreMiD. 
Categories: https://docs.premid.app/en/dev/presence/metadata#presence-categories

File: websites/D/Drama/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    },
+    "category": "socials & stream",
+    "logo": "https://cdn.discordapp.com/icons/730015520568639509/a_af5921a891d60ff6b22c79f0d44c7e3b.webp?size=512",
+    "thumbnail": "https://zyrex.xyz/storage/a2d579e2-55f2-44a3-81fc-bcae0ff389a6.png",
+    "version": "1.0.0",
+    "color": "#32f597",
+    "tags": [
+      "Forum",
+      "Social",
+      "Drama"
```
Comment (VocalZero) [inline websites/D/Drama/dist/metadata.json:16]: All tags should be written in lower case.

**Review by VocalZero — CHANGES_REQUESTED** [review]

**Comment by ririxidev** [comment]
Low quality presence *again*, creating new objects everytime, ESLint issues etc. Etc.

**Pull request closed without merging**